test_conformance/computeinfo/main.cpp

```diff
@@ -1422,10 +1422,12 @@ extern int test_extended_versioning(cl_device_id, cl_context, cl_command_queue,
                                     int);
 extern int test_device_uuid(cl_device_id, cl_context, cl_command_queue, int);
 
+extern int test_conformance_version(cl_device_id, cl_context, cl_command_queue,
+                                    int);
+
 test_definition test_list[] = {
-    ADD_TEST(computeinfo),
-    ADD_TEST(extended_versioning),
-    ADD_TEST(device_uuid),
+    ADD_TEST(computeinfo), ADD_TEST(extended_versioning), ADD_TEST(device_uuid),
+    ADD_TEST_VERSION(conformance_version, Version(3, 0))
```

Comment:
Pretty minor, but if you add a trailing comma here it stops clang-format from making the formatting of this list... unpleasant.

Reply:
+1, it would be nice to stick to one test per line